www/index.md

```diff
@@ -31,6 +31,21 @@ IE11 compatible
 By removing these arbitrary constraints htmx completes HTML as a 
 [hypertext](https://en.wikipedia.org/wiki/Hypertext)
 
+## HTML over the wire
+
+Serializing data to JSON on the server and parsing the JSON on the client is the way 
+many frameworks (Angular, React Vue) work today.
+
+But maybe there is a simpler way?
+
+Why not keep the client simple and stateless?
+
+htmx transfer html fragments and thus simplifies how modern web application
```

Comment:
```suggestion
htmx transfers server-rendered html fragments and thus simplifies how modern web applications
```

Once again, I'm trying to make things clearer for SPA devs. React already has a thing called a Fragment; some people got confused with intercooler because they didn't understand you were meant to use it with a server side app

also typo fix

Reply:
Thank you for this comment, I updated the file